lib/config/rules/jest.js

```diff
@@ -17,4 +17,16 @@ module.exports = {
   'jest/prefer-to-be-undefined': 'error',
   // Ensure `expect()` is called with a single argument and there is an actual expectation made.
   'jest/valid-expect': 'error',
+  // Suggest using expect.assertions() OR expect.hasAssertions()
+  'jest/valid-expect-in-promise': 'off',
```

Comment:
I think you are missing a rule (the above us not the correct description for the rule)

Reply:
Also, this rule should definitely be on, it’s a very common problem

Reply:
Good catch, it was missing https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-expect-assertions.md

Reply:
> Also, this rule should definitely be on, it’s a very common problem

Turned on in 23be4b8